docs/source/installation/versions.rst

```diff
@@ -2,22 +2,22 @@ Differences between Manim Versions
 ==================================
 
 While originally a single library, there are now three main versions of manim, 
-each with their own advantages, disadvantages and ideal use cases. 
+each with its own advantages, disadvantages, and ideal use cases. 
```

Comment:
Pretty sure "their" is correct

Reply:
It is more correct with 'its'

Reply:
Their.
There are many different versions. **They** all have advantages. They each come with **their** own advantages. Each **one** of them comes with **its** own set of advantages..."

Reply:
Ok that makes sense. I'm reverting back then.